src/encoded/static/components/item-pages/HiGlassViewConfigView.js

```diff
@@ -1104,17 +1141,17 @@ export const ConfirmModal = React.memo(function (props) {
                 {props.children || ''}
             </Modal.Body>
             <Modal.Footer>
-                <button type="button" onClick={handleConfirm} className="btn btn-success">
-                    <i className="icon icon-fw icon-check mr-05 fas" />{ confirmButtonText || 'OK' }
+                <button type="button" onClick={handleConfirm} className="btn btn-success" ref={(button) => { confirmButton = button; }}>
```

Comment:
Instead of making a ref function, best practice is to use [useRef](https://reactjs.org/docs/hooks-reference.html#useref) hook.